### ISSUE TITLE
Week 8 Crossover Grading - Kira

### ISSUE THREAD

**Issue by 4disease** [florayuan18/pikachudrinkingwindex#57]
| FRQ | Completed? | Deployed? | Documentation | Input| Points |
|----|----|----|----|----|----|
| 2 | Yes | [Yes](https://replit.com/@4DISEASE/CSATri2#Main.java) | [Link](http://blog.umbrellabox.cf/kirafrq/) | Yes | 1.5 |
| 3 | Yes | [Yes](https://replit.com/@4DISEASE/CSATri2#Main.java) | [Link](http://blog.umbrellabox.cf/kirafrq/) | - | 1 |
| 4 | Yes | [Yes](https://replit.com/@4DISEASE/CSATri2#Main.java) | [Link](http://blog.umbrellabox.cf/kirafrq/) | - | 1 |
| 5 | Yes | [Yes](https://replit.com/@4DISEASE/CSATri2#Main.java) | [Link](http://blog.umbrellabox.cf/kirafrq/) | - | 1 |
| 6 | Yes | [Yes](https://replit.com/@4DISEASE/CSATri2#Main.java) | [Link](http://blog.umbrellabox.cf/kirafrq/) | Yes | 1.5 |
| 7 | Yes | [Yes](https://replit.com/@4DISEASE/CSATri2#Main.java) | [Link](http://blog.umbrellabox.cf/kirafrq/) | - | 1 |
| 8 | Yes | [Yes](https://replit.com/@4DISEASE/CSATri2#Main.java) | [Link](http://blog.umbrellabox.cf/kirafrq/) | - | /1 |
| 9 | Yes | [Yes](https://replit.com/@4DISEASE/CSATri2#Main.java) | [Link](http://blog.umbrellabox.cf/kirafrq/) | - | /1 |
| 10 | Yes | [Yes](https://replit.com/@4DISEASE/CSATri2#Main.java) | [Link](http://blog.umbrellabox.cf/kirafrq/) | - | /1 |

**Comment by dsblack0** [comment]
### Comments per FRQ
| FRQ | Completed? | Deployed? | Documentation | Input? | FRQ Comments | Points |
| ---   | ---                |  ---             | ---                      | ---      | ---                      | ---       |
| 2 | Yes | Yes | Complete and thorough | Yes | I think it would have been helpful if explained what was being outputted when you asked for "Sq root first/second numbers" | |
| 3 | Yes | Yes | Complete and thorough | No |  It is a little confusing to understand what is happening when I just run this frq because none of the default input values are listed. Listing the default values on the runtime would help give context to the frq. | |
| 4 | Yes | Yes | Complete and thorough | Yes |  It would still be a little more helpful if you included something that allowed someone else to understand what was happening when they simply ran the frq but this one worked very well with giving the input and additional information on the longest streak. |
| 5 | Yes | Yes |  Complete and thorough | Yes for the first question | The invitation frq worked very well with the inputs, being able to change the address, and giving the final invite sentence for the output. For the password section of the frq, it was hard to follow what was happening on the runtime since you didn't include what the default values you used were |
| 6 | Yes | Yes | Complete and thorough | No |  The frq worked well and gave all of the desired outputs and it was much easier to follow as you gave the default values for the list of items sold as well. It would also have been helpful if you gave the default list of words for the first word select question as well |
| 7 | Yes | Yes | No reflection | No |  This frq worked well with the default values being clear and it was also easier to understand what was happening in the frq from your descriptions |
| 8 | Yes | Yes | No reflection  | No |  This frq worked well but it was very hard to follow with referencing back with the code because I wasn't able to see what the default data for the farm plots was. |
| 9 | Yes | Yes | No reflection | No |  This frq worked well and all of the outputs were clear to understand |
| 10 | No | No | No reflection | No |  I wasn't able to find your frq for Unit 10 |

### Overall Comments
- One of the main problems within your frqs was how you didn't include the default values that you used for the frqs that didn't include inputs. This caused it to be hard to follow what your code was doing when I only was looking at your runtime. Also, it would have been helpful if you overall just had more brief descriptions of what was happening in each frq. You had this in some but not all of your frqs.
- Your reflections all looked good and showed your understanding of the purpose and main concepts involved in each frq. I can tell that you have a good understanding of how you were implementing each frq question. You were missing reflections for the last 4 frqs, though.
- You were missing your frq for Unit 10

**Comment by 4disease** [comment]
## Reflection
- The benefit of having MVC as opposed to repl.it is that it shows more command of extra concepts (How to format HTML, how to take input on a website.) The downside to having MVC is that it may not be as responsive as a repl.it and the experience may be slower.
- Samhitha's documentation of each FRQ on her website was very good and showed the FRQs in context of the collegeboard problem. I could think about adding more context about the FRQ purpose to the repl.it runtime.
- Samhitha put input for most of the FRQs. Specifically FRQ 7 I found interesting - Her layout allowed for input of the second list that the first list would be compared to for duplicate usernames. A big positive of her FRQs is that she shows creativity about the inputs.
